package.json

```diff
@@ -138,6 +138,7 @@
     "@dendronhq/conventional-changelog-dendron": "^5.0.11",
     "chalk": "^4.1.1",
     "cross-env": "^7.0.3",
+    "eslint-plugin-neverthrow": "^1.1.4",
```

Comment:
see comment about this causing too much churn in the codebase. if we do this, would only run this on new code reviews

Reply:
regardless, would leave it as a separate pr and not combine with this one

Reply:
as long we cannot use typeaware eslint rules this neverthrow plugin has no effects. its now disabled because it results in Out of Memory error https://github.com/dendronhq/dendron/pull/3639/files#diff-e2954b558f2aa82baff0e30964490d12942e0e251c1aa56c3294de6ec67b7cf5R23
we can introduce the plugin some time in the future when typeaware linting works.